cirq/devices/noise_model.py

```diff
@@ -183,7 +183,11 @@ def _json_dict_(self):
 
 @value.value_equality
 class ConstantQubitNoiseModel(NoiseModel):
-    """Applies noise to each qubit individually at the start of every moment."""
+    """Applies noise to each qubit individually at the start of every moment.
+
+    This is the noise model that is wrapped around an operation when that
+    operation is given as "the noise to use" for a `NOISE_MODEL_LIKE` parameter.
```

Comment:
What is "the noise to use"?  Is this a specific parameter?  Can we use something more precise if possible.